.eslintrc.js

```diff
@@ -33,10 +33,10 @@ module.exports = {
         SwitchCase: 1,
       },
     ],
-    "linebreak-style": [
-      "error",
-      process.platform === "win32" ? "windows" : "unix",
-    ],
+    // "linebreak-style": [
+    //   "error",
+    //   process.platform === "win32" ? "windows" : "unix",
+    // ],
```

Comment:
Could you revert this? It's not ideal at the moment, but we'll need to find a proper fix to it.

using `--no-verify` to a commit should bypass the linting requirement on windows.

Reply:
sorry thought I undid that with my merges! have undone it in [6e88062](https://github.com/Together-100Devs/Together/pull/388/commits/6e88062d453ef1b54be70f6191096d15481a7013)